lychee-lib/src/extract/mod.rs

```diff
@@ -17,7 +17,18 @@ use plaintext::extract_plaintext;
 pub(crate) fn is_verbatim_elem(name: &str) -> bool {
     matches!(
         name,
-        "code" | "listing" | "plaintext" | "samp" | "script" | "textarea" | "xmp" | "pre"
+        "address"
+            | "code"
+            | "kbd"
+            | "listing"
```

Comment:
what is `listing` here? there is no `<listing>` tag.
P.S. I also cannot find the test case for it.

Reply:
It's obsolete. Didn't add a unit test because of that. It got entirely replaced by `pre` and `code` but there might be older sites out there which still use it.